tests/test_mongo.py

```diff
@@ -29,6 +29,7 @@
 from mongo_connector.connector import Connector
 from mongo_connector.util import retry_until_ok
 from mongo_connector.test_utils import (ReplicaSet,
+                                        ReplicaSetSingle,
```

Comment:
Refactored TestMongo to use ReplicaSetSingle when possible. A full ReplicaSet is only needed for `test_rollback`.